iroha/src/crypto.rs

```diff
@@ -28,7 +29,9 @@ pub struct KeyPair {
 }
 
 /// Public Key used in signatures.
-#[derive(Encode, Decode, Ord, PartialEq, Eq, PartialOrd, Debug, Clone, Hash, Default)]
+#[derive(
+    Copy, Encode, Decode, Ord, PartialEq, Eq, PartialOrd, Debug, Clone, Hash, Default, Deserialize,
+)]
 pub struct PublicKey {
     inner: [u8; 32],
```

Comment:
I think serialized data will look more pretty if we will use `#[serde(flatten)]` here
```suggestion
    #[serde(flatten)]
    inner: [u8; 32],
```

Reply:
1. we will replace serialization with hex
2. We do not have serialization to json right now

Reply:
@humb1t yeah, but we have deserialization from JSON config

Reply:
@vmarkushin yes - I understand, I thought that after https://jira.hyperledger.org/browse/IR-745 we will have two fields (inner + type) but hex representation will combine both fields inside one - so I will apply `flatten`. Thanks one more time.